### PR TITLE
Manual Merge

### PR DESCRIPTION
**Pull request by kunicmarko20**
Just approve

### CODE REVIEW

File: src/Form/Type/FormatterType.php [hunk 1]
```diff
@@ -58,18 +59,25 @@
...
+     */
+    private $toolbarManager;
+
     public function __construct(
         Pool $pool,
         TranslatorInterface $translator,
         ConfigManagerInterface $configManager,
         ?PluginManagerInterface $pluginManager = null,
-        ?TemplateManagerInterface $templateManager = null
+        ?TemplateManagerInterface $templateManager = null,
```
Comment (kunicmarko20) [inline src/Form/Type/FormatterType.php]: are we sure this should stay nullable on master?

**Comment by greg0ire** [comment]
Argh, FlintCI disapproves

File: src/Form/Type/FormatterType.php (hunk 1 above)
Comment (greg0ire) [reply src/Form/Type/FormatterType.php]: Maybe not, but this is not the PR to change it, don't do [evil merges](https://blog.joolsadams.org/2013/02/15/evil-merges-in-git/)

Comment (kunicmarko20) [reply src/Form/Type/FormatterType.php]: I know I know, I just now saw it and was wondering :smile:

**Comment by kunicmarko20** [comment]
Fixed.

**Pull request merged**